chips/nrf52832/src/interrupt_service.rs

```diff
@@ -1,21 +1,30 @@
-use crate::gpio;
-use nrf52::interrupt_service::InterruptService;
+use crate::deferred_call_tasks::DeferredCallTask;
+use nrf52::chip::Nrf52DefaultPeripherals;
 
-pub struct Nrf52832InterruptService {
-    nrf52: nrf52::interrupt_service::Nrf52InterruptService<'static>,
+/// This struct, when initialized, instantiates all peripheral drivers for the nrf52840.
+/// If a board wishes to use only a subset of these peripherals, this
+/// should not be used or imported, and a modified version should be
+/// constructed manually in main.rs.
+pub struct Nrf52832DefaultPeripherals<'a> {
+    pub nrf52_base: Nrf52DefaultPeripherals<'a>,
```

Comment:
Not sure we should name it `nrf52_base` as opposed to simply `nrf52` (which follows the name of the `nrf52` crate).

Reply:
done